File: 2_objc-for-swift-dev/section-1/Properties/Properties/ViewController.m
//
//  ViewController.m
//  Properties
//
//  Created by Suzy Lee on 7/28/20.
//  Copyright © 2020 Suzy Lee. All rights reserved.
//

#import "ViewController.h"
#import "Person.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    
    Person *person1 = [[Person alloc]init];
    person1.firstName = @"Suzy";
    [person1 setLastName: @"Lee"];
    
    
}


@end

File: 1_ios-dev-bootcamp/section-11/BMI-Calculator-iOS13-LayoutPractice-master/BMI-Calculator-LayoutPractice/Controllers/ViewController.swift
//
//  ViewController.swift
//  BMI-Calculator-LayoutPractice
//
//  Created by Angela Yu on 21/08/2019.
//  Copyright © 2019 App Brewery. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    @IBOutlet weak var heightLabel: UILabel!
    @IBOutlet weak var weightLabel: UILabel!
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view.
    }
    
    
    @IBAction func heightSliderChanged(_ sender: UISlider) {
        print("Current height: " + String(format: "%.1f", sender.value))
        heightLabel.text = String(format: "%.1f", sender.value)
    }
    
    
    @IBAction func weightSliderChanged(_ sender: UISlider) {
        print("Current weight: \(Int(sender.value))")
        weightLabel.text = String(Int(sender.value))
    }
    

}

